src/metalnx-web/src/main/java/com/emc/metalnx/controller/LoginController.java

```diff
@@ -147,6 +147,12 @@ public ModelAndView loginErrorHandler(final Exception e) {
 		 * https://github.com/irods-contrib/metalnx-web/issues/162
 		 */
 		Authentication auth = SecurityContextHolder.getContext().getAuthentication();
+		
+		// If this runs before login the auth will be an instance of 
+		// AnonymousAuthenticationToken. In that case we do not want to redirect to 
+		// "browse/home".  In addition getLoggedDataGridUser() returns the 
+		// "JargonException: IRODSAccount is null" because there is no longer an admin
+		// account logged in at startup so we need to skip that call.
```

Comment:
```suggestion
		// If this runs before login the auth will be an instance of
		// AnonymousAuthenticationToken. In that case we do not want to redirect to 
		// "browse/home".  In addition getLoggedDataGridUser() returns the
		// "JargonException: IRODSAccount is null" because there is no longer an admin
		// account logged in at startup so we need to skip that call.
```

Reply:
done